packages/kbn-apm-synthtrace/src/lib/apm/processors/get_transaction_metrics.ts

```diff
@@ -66,6 +66,10 @@ export function getTransactionMetrics(events: ApmFields[]) {
     'faas.id',
     'faas.coldstart',
     'faas.trigger.type',
+    'faas.billed_duration',
+    'faas.timeout',
+    'faas.coldstart_duration',
+    'faas.duration',
```

Comment:
these are metrics, not metadata fields. aggregating over them will likely cause an explosion of metricsets.